src/app/design/guidelines/content-containers/index.html

```diff
@@ -155,7 +155,28 @@ <h3 class="sky-subsection-heading">
   </h3>
 
   <p>
-    Popovers show small chunks of information that are only occasionally relevant to the user. If the content is relevant most of the time, it should be presented in-page instead. If the amount of content would require scrolling on a mobile device, use a flyout instead.
+    Popovers show small chunks of information that are only occasionally relevant to the user. Popovers are appropriate when:
```

Comment:
As explained above, I'd change the first sentence to something like "Popovers display small chunks of contextual content that do not always need to be visible."